src/util/exportToNDJson.js

```diff
@@ -0,0 +1,35 @@
+const { db } = require('./mongo');
+const supportedResources = require('../util/supportedResources');
+const fs = require('fs');
+
+/**
+ * Exports the list of resources included in the _type member of the request object to NDJson
+ * if the _type member doesn't exist it willl simply export everything included in the supportedResources list
+ * @param {Object} request http request object
+ * @param {*} clientId the response object
+ */
+const exportToNDJson = async (clientId, request) => {
+  let dirpath = './tmp/';
+  await fs.promises.mkdir(dirpath, { recursive: true });
+  let requestTypes;
+  if (request.query._type) {
+    requestTypes = request.query._type.split(','); //this is the list types to export
+  } else {
+    //create list of requested types if request.query._type param doesn't exist
+    requestTypes.push(supportedResources);
```

Comment:
I'm a little confused why this is a push if it's initially creating the requestTypes list

Reply:
Could potentially do this whole block as a `||` with "optional chaining" (safe operator) to avoid the empty requestTypes declaration?
something like `requestTypes = request.query._type?.split(',') || supportedResources`
or could use a ternary operator